pkg/packagekit/updates.jsx

```diff
@@ -460,7 +413,9 @@ class ApplyUpdates extends React.Component {
                     <td>{action.package}</td>
                 </tr>));
         } else {
-            actionHTML = _("Initializing...");
+            return <EmptyStatePanel title={ _("Initializing...") }
```

Comment:
With this return I find the `render()` method now has a weird structure. Can you please invert the condition, i.e. at the top of the method do

```
if (this.state.actions.length === 0)
    return <EmptyStatePanel ...>;
```

and then move `actionHTML` and `logRows` to be `const`? I'd even eliminate `actionHTML` entirely and move the two elements right into the main JSX below, then it's easier to see at a glance how the page looks like.